.rubocop.yml

```diff
@@ -14,7 +14,7 @@ Metrics/LineLength:
 
 Metrics/MethodLength:
   CountComments: false
-  Max: 26
+  Max: 50
```

Comment:
Actually, the rubocop is skipping the comments, but not the log messages, tha is why I had to make it bigger.

Do you know if it is possible to add this kind of exception to rubocop?

The `namespace_handler`or `parse_namespace` has a lot of log lines that are being accounted as normal code lines.

I can bump it down back to the previous value, but it would cause some methods to be splitted unnecessarily.